spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/store/jwk/JwkTokenStore.java

```diff
@@ -86,16 +88,23 @@
  * @author Joe Grandja
  */
 public final class JwkTokenStore implements TokenStore {
-	private final JwtTokenStore delegate;
```

Comment:
Is that true? Couldn't another token store do the exact same thing? Remember we don't allow consumers to set this value anyways, so this is of no risk but keeps the remaining code as loosely coupled as possible.